src/routes/index.ts

```diff
@@ -519,6 +519,15 @@ router.route(config.UPDATE_MANAGE_TRUSTS_ORCHESTRATOR_URL)
   )
   .all(updateManageTrustsOrchestrator.handler);
 
+router.route(config.UPDATE_MANAGE_TRUSTS_ORCHESTRATOR_URL + config.TRUST_ID)
```

Comment:
could be more explicit about the path by adding a "/change_handler/" or something before the trust_id?

Reply:
I guess "reviewed-trust" is what it is recording?

Reply:
Yeah happy to do that, will do